app/assets/javascripts/accounts.coffee

```diff
@@ -1,3 +1,9 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+$(document).ready ->
+  $('.ui.checkbox').click ->
+    if($(@).hasClass('checked'))
```

Comment:
Parece que é o equivalente ao this do jquery, mas não tenho certeza realmente. Na pesquisa que eu fiz dizia isso, e como não sei, só segui.